pkgs/servers/apache-directory-server/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchzip, adoptopenjdk-jre-openj9-bin-11, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "apache-directory-server";
```

Comment:
This file be pname

Reply:
Thank you for the suggestion. From my understanding, `pname` will ignore the version field in the full package name. Why is this desired in this case?

Reply:
No, it is the other way around. name is constructed from pname + version and if set name it overwrites them.